### ISSUE TITLE
A feature that helps pin manga in library

### ISSUE THREAD

**Issue by BuriBuriZymon** [mihonapp/mihon#211]
### Describe your suggested feature

So this is a feature from a manga app in iOS called Aidoku.
What this does is it pins the manga with unread chapters on top.

### Other details

I don't have any iOS device anymore so unfortunately I can't share a SS...sorry.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
You can filter by unread in your library view, just tap the filter icon at the top right and select it

**Comment by BuriBuriZymon** [comment]
Ik and doing that keeps the unread manga above the read manga but then it obviously doesn't show my last read manga first😭

In Aidoku(iOS manga app) turning on that option lets you do both of those things together.
![Uploading IMG_1475.png …]()
Found this in their discord😭 
It separates the read and unread manga.

**Comment by not-fried-dumplings** [comment]
You can use filters and also sort by last read. That is what I do, so all your recently read manga will all be on top.

**Comment by MajorTanya** [comment]
> Ik and doing that keeps the unread manga above the read manga but then it obviously doesn't show my last read manga first😭

No, I'm talking about using the Unread Filter (which will only show entries with unread chapters). This can be combined without issue with the "Last Read" Sort order to get the result you seem to be describing.

**Comment by not-fried-dumplings** [comment]
Your screenshot doesn't really say much. Are the top manga the ones you've read recently? If so, you can do that with the sort option.

**Comment by BuriBuriZymon** [comment]
Let me explain it clearly.
Those manga in the top are sorted by last read.
And manga with zero unread chapters get a separate section in the screen as you can see in the SS.

If I use the unread filter in Mihon it hides the unread manga and that is not what I'm looking for.


Well it's just a request and if they can't do it then it's fine lol

Ty for your time tho

**Comment by not-fried-dumplings** [comment]
I don't think you can have both of them at once in Mihon. You can have the 2 options separately tho, or is that not to your liking?
